### PR TITLE
Added keyboard logic

### PR DESCRIPTION
**Pull request by amiryonatan**

### CODE REVIEW

File: packages/wix-ui-core/src/components/Dropdown/Dropdown.spec.tsx [hunk 1]
```diff
@@ -9,7 +9,6 @@ describe ('Dropdown', () => {
   const options = [1, 2, 3, 4, 5].map(x => ({
     id: x,
     value: `value${x}`,
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Dropdown/Dropdown.spec.tsx]: what heppens if value is a custom value (some node with Icon) and it get's selected. What will be displayed in the Input? Maybe the displayname was a good approach, or you can have valueParser func that you get from the user to know waht to display on select

File: packages/wix-ui-core/src/components/Dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -3,18 +3,24 @@ import Popover, {SharedPopoverProps} from '../Popover';
...
+import {Option} from './DropdownContent/DropdownContent';
 import {CLICK, CLICK_TYPE, HOVER, HOVER_TYPE, SINGLE_SELECT, SINGLE_SELECT_TYPE, MULTI_SELECT, MULTI_SELECT_TYPE, SEPARATOR} from './constants';
 
+type DropdownClasses = {
+  targetElement: string;
+};
+
 interface DropdownProps {
-  children: () => React.ReactNode;
+  children: (onKeyDown) => React.ReactNode;
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Dropdown/Dropdown.tsx]: we talked today that maybe having an object is better that only one param

File: packages/wix-ui-core/src/components/Dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -94,7 +105,37 @@ class Dropdown extends React.PureComponent<DropdownProps & SharedDropdownProps,
     }
   }
 
-  _onOptionClick(option, evt) {
+  onKeyDown(evt: React.KeyboardEvent<HTMLElement>) {
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/Dropdown/Dropdown.tsx]: Please remember to talk about it with Ben, to make sure we are synced

File: packages/wix-ui-core/src/components/Dropdown/Dropdown.spec.tsx (hunk 1 above)
Comment (amiryonatan) [reply packages/wix-ui-core/src/components/Dropdown/Dropdown.spec.tsx]: 👍

**Pull request merged**